calculator/src/main/java/edu/temple/cis3296/CalculatorViewGui.java

```diff
@@ -35,7 +35,7 @@
 	 * Creates the panels and components for the JFrame
 	 */
```

Comment:
This change looks good, but I think you should add a test to check it!

Reply:
Unit tests are not appropriate here, since this is just a GUI label. Running the program and observing the correct title in place should be sufficient. Thank you for your feedback!

Reply:
Good point! You're right.